editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -2290,6 +2365,11 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	split_sheet_offset_hb->add_child(split_sheet_offset_vb);
 	split_sheet_settings_vb->add_child(split_sheet_offset_hb);
 
+	Button *auto_slice = memnew(Button);
+	auto_slice->set_text(TTR("Auto-Slice"));
```

Comment:
Note that in the region editor it's without a dash, we should be consistent about it.
https://github.com/godotengine/godot/blob/e63c40e59c6650bdba02c2ceff3390ee9273ac46/editor/plugins/texture_region_editor_plugin.cpp#L1139
```suggestion
	auto_slice->set_text(TTR("Auto Slice"));
```

Reply:
Done, in next push.